src/main/java/utils/ExportResultsUtil.java

```diff
@@ -1,50 +1,49 @@
 package utils;
 
 import refactoring.Refactoring;
+
 import javax.swing.*;
-import java.io.File;
+import java.awt.*;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.nio.file.Paths;
 import java.nio.file.StandardOpenOption;
 import java.util.List;
+
 import static java.nio.file.Files.createFile;
 import static java.nio.file.Files.write;
 
 /**
- * Class exports (MoveMethod) refactoring results to the file
+ * Exports refactoring results to the file
  */
 public class ExportResultsUtil {
 
     /**
-     * Exports refactoring results to the file.
+     * @param refactorings list of refactoring suggestions.
+     * @param panel panel of current project.
      */
-    public static void export(List<? extends Refactoring> refactorings) {
-        final JFileChooser fileChooser = new JFileChooser();
-        fileChooser.setFileSelectionMode(JFileChooser.DIRECTORIES_ONLY);
-        int returnVal = fileChooser.showOpenDialog(null);
-        if (returnVal == JFileChooser.CANCEL_OPTION) {
+    public static void export(List<? extends Refactoring> refactorings, JPanel panel) {
+        JFrame frame = (JFrame) SwingUtilities.getWindowAncestor(panel);
+        FileDialog fileDialog = new FileDialog(frame, "Results export", FileDialog.SAVE);
```

Comment:
Add title of the dialog in IntelliJDeodorantBundle.properties
` export.title = Export`
Then you can get the title using ` IntelliJDeodorantBundle.message("export.dialog")`